blocks/library/latest-posts/index.php

```diff
@@ -13,13 +13,18 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts( array(
+	$args = array(
 		'numberposts' => $attributes['postsToShow'],
 		'post_status' => 'publish',
 		'order'       => $attributes['order'],
 		'orderby'     => $attributes['orderBy'],
 		'category'    => $attributes['categories'],
-	) );
+	);
+	$instance = array();
```

Comment:
Take a look at https://github.com/WordPress/wordpress-develop/blob/ba195e435339813afc931db6359a515b92142292/src/wp-includes/widgets/class-wp-widget-recent-posts.php#L135-L137

You could use those values as the default `$instance`, except for `number` which should be `$attributes['postsToShow']`.

Reply:
looks like https://github.com/WordPress/wordpress-develop/blob/ba195e435339813afc931db6359a515b92142292/src/wp-includes/widgets/class-wp-widget-recent-posts.php#L51 might be a problem for title, even in the latest force update.

Reply:
I also wonder if title shouldn't be changed to a constant of 'Latest posts'. But then if an existing plugin matches on 'Recent Posts' (fragile design), then it's not parity.